### PR TITLE
DOCKER-255 add share on top of 'alfresco-enterprise-repository' image

### PR DESCRIPTION
**Pull request by tgeens**
Alternative for https://github.com/xenit-eu/docker-alfresco/pull/5/

### CODE REVIEW

**Comment by anghelutar** [comment]
This is what we discussed in the morning, but with a separate build.gradle so that the main build.gradle does not increase further. Plus, the whole folder legacy can be removed at the right time.
Build time of the project on Jenkins is longer, but the project is cleaner.

Note: for community we need to add alfresco-share-services amp to dependencies.

File: build.gradle [hunk 1]
```diff
@@ -28,6 +28,14 @@ subprojects {
     if (project.name == "docker-init-script") {
         return
     }
+
+
+    if (project.path.startsWith(':legacy')) {
+        return
+    }
+
+    println "root build.gradle: configuring ${project.name}"
```
Comment (vierbergenlars) [inline build.gradle]: Use logger.info or logger.debug instead of `println`

Comment (vierbergenlars) [inline build.gradle]: Use `logger.info` or `logger.debug` instead of `println`

Comment (anghelutar) [reply build.gradle]: Yes, there is a ticket for this: https://xenitsupport.jira.com/browse/DOCKER-256 (piling up)

Comment (tgeens) [reply build.gradle]: I'm just going to remove that line, it was for debugging purposes

**Comment by tgeens** [comment]
Build times are a concern.

Is it worth putting this in a separate repo `docker-alfresco-legacy` ?

**Comment by anghelutar** [comment]
Imo it is only worth to separate out once we solve the cascading dependencies somehow.

**Comment by tgeens** [comment]
Ethias needs 6.1 also

**Comment by anghelutar** [comment]
@tgeens is this ok for you? (asked @vierbergenlars if he could finish the implementation, but he's leaving on holiday and was not sure he had time)

**Comment by tgeens** [comment]
@anghelutar Looks good to me. 

Cleaned up the remaining println statements, so can be merged if it builds.

File: build.gradle [hunk 2]
```diff
@@ -130,7 +131,7 @@ subprojects {
                 buildArgs << project.extraBuildArgs
 
             doFirst {
-                println buildArgs
+                logger.info buildArgs
```
Comment (vierbergenlars) [inline build.gradle]: You can not `logger.info` a map.

Also, I think this was just for debugging, so it can be removed

Comment (tgeens) [reply build.gradle]: Removed

**Pull request merged**